### DISCUSSION TITLE
how do i use it?

### DISCUSSION THREAD

**Discussion by ghost** [StarGate01/AmongUs-Mumble#63] [Q&A]
sorry im really dumb with computers. how do I use it? i barely know how extract a zip file qwq

**Comment by StarGate01** [comment]
Hey, please join our Discord server where we have detailed tutorials and a help channel: https://discord.gg/4UkHEJ5sqg

Basic instruction can also be found on our wiki: https://github.com/StarGate01/AmongUs-Mumble/wiki/Installation-and-Setup
